src/App.js

```diff
@@ -887,7 +888,7 @@ class App extends React.Component<Props, State> {
   onEquipmentSelected = (placeInfoId: string, equipmentInfo: EquipmentInfo) => {
     this.props.routerHistory.replace('equipment', {
       id: placeInfoId,
-      eid: equipmentInfo._id,
+      eid: get(equipmentInfo, 'properties._id'),
```

Comment:
0: In EquipmentInfo.js:91 we still say that there is an _id on the top level, maybe remove that?

Reply:
Yup! :)

Reply:
https://trello.com/c/rZ3BYj1K/144-fix-equipmentinfo-type-id-in-wheelmap-react-frontend